packages/button/src/react/index.tsx

```diff
@@ -128,29 +128,33 @@ interface ButtonStatics {
   iconAligns: typeof vars.iconAligns
   sizes: typeof vars.sizes
 }
-
 interface BaseButtonProps {
   appearance?: ValueOf<typeof vars.appearances>
+  // TODO: rm? dup of html attr
   disabled?: boolean
```

Comment:
i think only button has `disabled` - so you'd want to leave it so the anchors can also be disabled

Reply:
good call